capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -180,21 +178,28 @@ bool ExecuteTaskSolutionCapability::constructMotionPlan(const moveit_task_constr
 		}
 		exec_traj.trajectory = std::make_shared<robot_trajectory::RobotTrajectory>(model, group);
 		exec_traj.trajectory->setRobotTrajectoryMsg(state, sub_traj.trajectory);
+		// Check that sub trajectories that contain a valid trajectory has controllers configured
+		if (!sub_traj.trajectory.joint_trajectory.points.empty() && sub_traj.execution_info.controller_names.empty()) {
+			RCLCPP_WARN(LOGGER,
+			            "The trajectory of stage '%i' from task '%s' does not have any controllers specified for "
+			            "execution. This might lead to weird behavior.",
```

Comment:
Definitely be more specific here.

```suggestion
			            "trajectory execution. This might lead to unexpected controller selection.",